src/ert/gui/simulation/run_dialog.py

```diff
@@ -48,6 +49,7 @@ class RunDialog(QDialog):
     def __init__(
         self,
         config_file: str,
+        ert: EnKFMain,
```

Comment:
Can we maybe get rid of the dependency on EnKFMain here altogether? :thinking: As far as I can see it is only used to determine if the plot button should be enabled or not:
https://github.com/equinor/ert/blob/1ef3bbf82c91aaf10007c8466c94f2d0dfbda62e/src/ert/gui/simulation/run_dialog.py#L118
 But I cannot see any situation where this would evaluate to false.